_overviews/scala3-book/scala-tools.md

```diff
@@ -38,7 +38,20 @@ $ mkdir hello
 $ cd hello
 ```
 
-Then create a file named _build.sbt_ that contains this line:
+In the directory `hello`, create a subdirectory `project`:
+
+```bash
+$ mkdir project
+```
+
+Create a file named _build.properties_ in the directory `project`, with
+the following content:
+
+```text
+sbt.version=1.5.4
```

Comment:
I think it’s important to teach people to set the version of sbt, otherwise loading the build is undeterministic.

Reply:
Yes, good point!